templates/paloalto_panos_test_security-policy-match.textfsm

```diff
@@ -0,0 +1,36 @@
+Value Filldown NAME (.*?)
+Value INDEX (\S+)
+Value Required FROM (\S+)
+Value SOURCE (\S+)
+Value SOURCE_REGION (\S+)
+Value TO (\S+)
+Value DESTINATION ([\S+\s+]+)
+Value DESTINATION_REGION (\S+)
+Value USER (\S+)
+Value SOURCE_DEVICE (\S+)
+Value DESTINATION_DEVICE (\S+)
+Value CATEGORY (\S+)
+Value APPLICATION_SERVICE (\[.*\])
+Value APPLICATION_SERVICE_IMPLICIT (\[.*\])
+Value ACTION (\S+)
+Value ICMP_UNREACHABLE (\S+)
+Value TERMINAL (\S+)
+
+Start
+  ^\"${NAME};\s+index:\s+${INDEX}\"\s+\{
```

Comment:
I think just grabbing the index should suffice rather than escaping the characters after, but you can leave it how it is as well. Just a nit pick on my part.